prespective_rectification/utils.py

```diff
@@ -0,0 +1,112 @@
+import numpy as np
+
+
+def convert_theta_to_two_points(x, y, theta):
+    x2 = x + 1000 * np.cos(theta)
+    y2 = y + 1000 * np.sin(theta)
+    x1 = x - 1000 * np.cos(theta)
+    y1 = y - 1000 * np.sin(theta)
+    line = [x1, y1, x2, y2]
+    return line
```

Comment:
Can return right away:
```suggestion
    return (x1, y1, x2, y2)
```

Reply:
hahaha yeah